cfme/tests/cloud_infra_common/test_html5_vm_console.py

```diff
@@ -171,9 +171,15 @@ def _validate_login():
             time.sleep(15)
 
         # create file on system
-        vm_console.send_keys("touch blather\n")
-        logger.info("Wait 5 sec for file creation")
-        time.sleep(5)  # Wait for file creation
+        def _touch_file():
+            vm_console.send_keys("\n\ntouch blather\n")
```

Comment:
I think I should reword the log entry, actually I am not waiting for the file creation, but giving time for send_keys to do the work and then run the check using find_text_on_screen, because if we take image of console too early, it might not see all characters being sent, even if they were sent and received correctly by the console.